src/dos/drives.h

```diff
@@ -54,11 +54,26 @@ class DriveManager {
 	static void CycleDisks(int drive, bool notify);
 	static void CycleAllDisks(void);
 	static char *GetDrivePosition(int drive);
+	static void SetDiskSpeed(DiskSpeed disk_speed, DiskType disk_type);
+	static void RegisterIoCallback(std::function<void()> callback,
+	                               DiskType disk_type);
```

Comment:
Blank lines before and after function definitions that span two lines; this is very hard to read.